src/measure_theory/integral/set_integral.lean

```diff
@@ -448,6 +448,14 @@ lemma set_integral_mono_set (hfi : integrable_on f t μ) (hf : 0 ≤ᵐ[μ.restr
   ∫ x in s, f x ∂μ ≤ ∫ x in t, f x ∂μ :=
 integral_mono_measure (measure.restrict_mono_ae hst) hf hfi
 
+lemma set_integral_const_le {c : ℝ} (hs : measurable_set s) (hμs : μ s ≠ ∞)
```

Comment:
That name does not fit the result. When I read it, I expect a lemma about the set integral of a constant being less than something. Maybe `set_integral_ge_of_const_le`?